spec/unit/chunk_insert_spec.rb

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2011 - 2013, SoundCloud Ltd., Rany Keddo, Tobias Bielohlawek, Tobias
+# Schmidt
+
+require File.expand_path(File.dirname(__FILE__)) + '/unit_helper'
+
+require 'lhm/chunk_insert'
+
+describe Lhm::ChunkInsert do
+  before(:each) do
+    @origin = Lhm::Table.new('foo')
+    @destination = Lhm::Table.new('bar')
+  end
+
+  describe "#sql" do
+    describe "when migration has no conditions" do
+      before { @migration = Lhm::Migration.new(@origin, @destination) }
+
+      it "makes uses a simple where clause" do
```

Comment:
i really hate to nitpick the grammar of the name of a test, but... there's an extra word here 😄

Reply:
fixed!